go/oasis-node/cmd/debug/fixgenesis/fixgenesis.go

```diff
@@ -137,14 +222,64 @@ func doFixGenesis(cmd *cobra.Command, args []string) {
 	}
 }
 
+func convertRuntime(rt *oldRuntime) *registry.Runtime {
+	crt := &registry.Runtime{
+		ID:              rt.ID,
+		EntityID:        rt.EntityID,
+		Genesis:         rt.Genesis,
+		Kind:            rt.Kind,
+		TEEHardware:     rt.TEEHardware,
+		Version:         rt.Version,
+		KeyManager:      rt.KeyManager,
+		Staking:         rt.Staking,
+		TxnScheduler:    rt.TxnScheduler,
+		GovernanceModel: registry.GovernanceEntity,
+		Executor: registry.ExecutorParameters{
+			GroupSize:         rt.Executor.GroupSize,
+			GroupBackupSize:   rt.Executor.GroupBackupSize,
+			AllowedStragglers: rt.Executor.AllowedStragglers,
+			RoundTimeout:      rt.Executor.RoundTimeout,
+			MaxMessages:       32, // TODO: What should this be set to?
```

Comment:
Good question, so setting this also requires configuring the global roothash consensus parameter (`MaxRuntimeMessages`). Either fixgenesis should set that as well (to the same value?) or we leave this at zero. Let's set this to 32 (both per-runtime and the global roothash consensus parameter) and then we can discuss what to set this to before the actual upgrade.